notewhacker.lisp

```diff
@@ -96,7 +96,17 @@ element."
     :accessor midi-channel :initarg :midi-channel :initform nil
     :documentation "Events with matching channel information target
     this staff. If nil, all events match. Used for matching Midi
-    events to staff."))
+    events to staff.")
+   (min-base-key
+    :accessor min-base-key :initarg :min-base-key :initform nil
+    :documentation "Minimum base key for the staff. Notes will be
+    created in `%create-new-targets` between min-base-key and
+    max-base-key")
+   (max-base-key
+    :accessor max-base-key :initarg :max-base-key :initform nil
+    :documentation "Maximum base key for the staff. Notes will be
+    created in `%create-new-targets` between min-base-key and
```

Comment:
I've tried to use older Emacs' symbol-referencing style in docstrings when referring to functions or symbols; that is, `` `%create-new-targets'`` instead of `` `%create-new-targets` ``. They're then nicely formatted in Emacs buffer (even though e.g. Slime inspector won't make links of them, as one might wish based on native Emacs' docstring symbol-reference handling).

(More information: https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html (search for "older single-quoting convention"))

```suggestion
    created in `%create-new-targets' between min-base-key and
```